README.md

```diff
@@ -2,29 +2,74 @@
 
 # upgrade-schedule-operator
 
-This is a upgrade-schedule-operator repository containing some basic files every repository
-needs.
-
-To use it just hit `Use this upgrade-schedule-operator` button or [this link][generate].
-
-Things to do with your newly created repo:
-
-1. Run`devctl replace -i "upgrade-schedule-operator" "$(basename $(git rev-parse
-   --show-toplevel))" --ignore '.git/**' '**'`.
-2. Run `devctl replace -i "upgrade-schedule-operator" "$(basename $(git rev-parse
-   --show-toplevel))" --ignore '.git/**' '**'`.
-3. Go to https://github.com/giantswarm/upgrade-schedule-operator/settings and make sure `Allow
-   merge commits` box is unchecked and `Automatically delete head branches` box
-   is checked.
-4. Go to https://github.com/giantswarm/upgrade-schedule-operator/settings/access and add
-   `giantswarm/bots` with `Write` access and `giantswarm/employees` with
-   `Admin` access.
-5. Add this repository to https://github.com/giantswarm/github.
-6. Create quay.io docker repository if needed.
-7. Add the project to the CircleCI:
-   https://circleci.com/setup-project/gh/giantswarm/upgrade-schedule-operator
-8. Change the badge (with style=shield):
-   https://circleci.com/gh/giantswarm/upgrade-schedule-operator.svg?style=shield&circle-token=TOKEN_FOR_PRIVATE_REPO
-   If this is a private repository token with scope `status` will be needed.
-
-[generate]: https://github.com/giantswarm/upgrade-schedule-operator/generate
+This operator is intended to automate scheduling, triggering and notification for upgrades of giant swarm workload clusters.
+
+## prerequisites
+
+Running this operator alone will enable the automatic scheduling and triggering of upgrades on a management cluster in any provider.
```

Comment:
`upgrades on a management cluster`

Is that correct? It's not upgrading the management cluster is it? Or is this referring to triggering a job that runs on the management cluster to perform the upgrade?

Reply:
Uh yeah maybe the wording here is back. basically what I mean is, if you add the controller to your MC you can upgrade the WCs on it. I'll try to word it better.